index.html

```diff
@@ -23,46 +22,48 @@
             <div class="wrapper">
                 <nav class="nav-bar">
                     <ul>
-                        <li class="logo"><a href="#">My Logo</a></li>
-                        <li class="menu-desktop"><a href="#">Portfolio</a></li>
-                        <li class="menu-desktop"><a href="#">About</a></li>
-                        <li class="menu-desktop"><a href="#">Contact</a></li>
+                        <li class="logo"><a href="#" tabindex="1">My Logo</a></li>
+                        <li class="menu-desktop"><a href="#" tabindex="2">Portfolio</a></li>
+                        <li class="menu-desktop"><a href="#" tabindex="3">About</a></li>
+                        <li class="menu-desktop"><a href="#" tabindex="4">Contact</a></li>
                     </ul>
-                    <div class="menu">
-                            <div class="toogle-btn"></div>
-                            <div class="toogle-btn"></div>
-                            <div class="toogle-btn"></div>
+                    <div class="menu" tabindex="2">
+                        <div class="toogle-btn"></div>
+                        <div class="toogle-btn"></div>
+                        <div class="toogle-btn"></div>
                     </div>
                 </nav>
             </div>
         </header>
         <section class="welcome-section">
             <div class="headline-wrapper">
                 <div class="headline-container">
-                    <h2 class="intro-text">I’m Clinton<br> Glad to see you!</h2>
-                    <p class="intro-message">I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <h2 class="intro-text">I’m Clinton <br /> Glad to see you!</h2>
```

Comment:
- [ ] I think there is an issue because you are using the `<h1>` tag for the "Contact section" and since this is the most important `<hX>` tag (semantically speaking) there can only be one per page and it should be used in the most important title. For this reason, I would suggest using the `<h1>` tag instead of the `<h2>` for this title because it's the most important. Also, you could use the `<h2>` for the section titles and the `<h3>` for less important titles (e.g., project titles). I think this structure could be a good one for your portfolio :wink: :pushpin:

Reply:
thank you for your review.